tls_codec/src/quic_vec.rs

```
@@ -129,6 +132,7 @@ pub fn write_variable_length(content_length: usize) -> Result<Vec<u8>, Error> {
     }
     let mut len = content_length;
     let l = length_bytes.len();
+    hax_lib::fstar!("admit ()"); // underflows
```

Here we should also really return an error.